### ISSUE TITLE
Off-by-one issue in the DataFrame - Last line is not shown in dataframe component

### ISSUE THREAD

**Issue by thondeboer** [streamsync-cloud/streamsync#188]
I noticed that the last line in a dataframe is missing when you try to scroll down...

```python
import streamsync as ss
import pandas as pd

initial_state = ss.init_state({
    "df": pd.DataFrame(
        {
            "col_a": list(range(11)),
            "col_b": list(range(11,0,-1))
        }
    )
})
```
This is the dataframe showing scrolled down to the bottom...Only row 10 is shown (index 9), not 11 (index 10)

![image](https://github.com/streamsync-cloud/streamsync/assets/3422829/6bcf4806-00ff-4af3-9cf1-f0e6ed9b4a5b)

I tracked it down to this statement in CoreDataframe.vue

```javascript
const rowOffset = computed(() => {
	let maxOffset: number;
	if (fields.wrapText.value == "yes") {
		maxOffset = rowCount.value - 1;
	} else {
		maxOffset = rowCount.value - displayRowCount.value;
	}
	const newOffset = Math.min(
		Math.floor(relativePosition.value * maxOffset),
		maxOffset,
	);
	return newOffset;
});
```

Math.floor should be Math.ceil.

This works:

```javascript
const rowOffset = computed(() => {
	let maxOffset: number;
	if (fields.wrapText.value == "yes") {
		maxOffset = rowCount.value - 1;
	} else {
		maxOffset = rowCount.value - displayRowCount.value;
	}
	const newOffset = Math.min(
		Math.ceil(relativePosition.value * maxOffset),
		maxOffset,
	);
	return newOffset;
});
```

relativePostion.value never reaches 1, so always maxOffset -1

**Comment by FabienArcellier** [comment]
Thanks for the test case and the fix. It was a great help. I have reproduced the bug on firefox. I was confused first because the component was working fine on chrome / chromium.

I will open the PR.

**Comment by ramedina86** [comment]
Fixed in v0.3.0
